README.md

```
@@ -440,6 +442,168 @@ print(f"Tokens: {result.tokens_used}")
+```
+
+---
+
+### Multi-Query Retrieval
+
+For **oblique queries** — where the relevant documents don't share surface vocabulary
+with the query — a single BM25 pass misses most of the relevant corpus. The
+multi-query pattern expands coverage by reformulating the query N times with diverse
+vocabulary before retrieval, then merging and reranking the union.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)